girder/api/describe.py

```diff
@@ -218,6 +245,87 @@ def param(self, name, description, paramType='query', dataType='string',
         self._params.append(param)
         return self
 
+    def modelParam(self, name, description=None, model=None, destName=None, paramType='path',
+                   plugin='_core', level=None, required=True, force=False, exc=True,
+                   requiredFlags=None, **kwargs):
+        """
+        This should be used in lieu of ``param`` if the parameter is a model ID
+        and the model should be loaded and passed into the route handler.
+
+        :param name: The name passed in via the request, e.g. 'id'.
+        :type name: str
+        :param description: The description of the parameter. If not passed, defaults
+            to "The ID of the <model>."
```

Comment:
It uses the `model` param, not the name, so it would say "The ID of the folder" in that case.